src/library_webgl.js

```diff
@@ -550,7 +562,14 @@ var LibraryGL = {
 
       var ctx = 
 #if USE_WEBGL2
-        (webGLContextAttributes.majorVersion > 1) ? canvas.getContext("webgl2", webGLContextAttributes) :
+        (webGLContextAttributes.majorVersion > 1)
+        ?
+#if MIN_CHROME_VERSION <= 57
+          !(getChromeVersion() <= 57) && canvas.getContext("webgl2", webGLContextAttributes)
```

Comment:
If `USE_WEBGL2`, `webGLContextAttributes.majorVersion > 1`, `MIN_CHROME_VERSION > 57`, and `getChromeVersion() <= 57` then this works out to `var ctx = false`?

```
> true ? !true && 2 : 1
false
```

Reply:
Hmm, if `MIN_CHROME_VERSION > 57`, then this block won't be included, so you probably meant `MIN_CHROME_VERSION <= 57`.

Yeah, in that case that's right, on Chrome <= 57, `ctx` will be `false`, and in below the check `if (!ctx)` will trigger and report WebGL 2 to be unsupported.

Reply:
Oh, gotcha. 👍